Cargo.toml

```diff
@@ -321,7 +321,7 @@ reth-ethereum-forks = { path = "crates/ethereum-forks" }
 reth-ethereum-payload-builder = { path = "crates/ethereum/payload" }
 reth-etl = { path = "crates/etl" }
 reth-evm = { path = "crates/evm" }
-reth-evm-ethereum = { path = "crates/ethereum/evm" }
+reth-evm-ethereum = { path = "crates/ethereum/evm", default-features = false }
```

Comment:
given that, we don't enable the `std` feature anywhere in the project. @mattsse we should propagate this change to all crates that use `reth-evm-ethereum` as a dependency, so they enable the `std` feature of it, right?

Reply:
Maybe I'm missing something, but I'm not sure how `reth-evm-ethereum` depends on `c-kzg`. Could you please clarify that for me?

Reply:
The following dependencies in `reth-evm-ethereum` have `c-kzg` as default features:

1. `reth-primitives`
2. `reth-revm`

AFAIC, it seems that these are the main culprits; although, the rabbit hole may go deeper. Would the move be to go from `c-kzg` -> `kzg-rs` in the `reth-primitives` and `reth-revm`?